### PR TITLE
Fix services command injection

### PR DESCRIPTION
**Pull request by murrant**
Reported by 
● Darek Jensen
● haxmeadroom

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Review by haxmeadroom — COMMENTED** [review]
It would probably also help to sanitize `service_ip, hostname, and service_param`

**Comment by Jellyfrog** [comment]
github doesnt allow you to comment on lines too far outside a diff, but on line 135 this is wrong also, should be an array with arguments

**Comment by murrant** [comment]
@Jellyfrog absolutely...  unfortunately, service_params is just a string stored in the database free-form entered by the admin.   My thoughts is change it to accept both that and array based parameters (when running, not saving). But that also requires a ui change.

File: includes/services.inc.php [hunk 1]
```diff
@@ -120,17 +121,21 @@ function poll_service($service)
...
     $old_status = $service['service_status'];
+    $service['service_type'] = Clean::fileName($service['service_type']);
+    $service['service_ip'] = Clean::fileName($service['service_ip']);
+    $service['hostname'] = Clean::fileName($service['hostname']);
+    $service['overwrite_ip'] = Clean::fileName($service['overwrite_ip']);
     $check_cmd = '';
 
     // if we have a script for this check, use it.
-    $check_script = Config::get('install_dir') . '/includes/services/check_' . strtolower($service['service_type']) . '.inc.php';
+    $check_script = Config::get('install_dir') . '/includes/services/check_' . strtolower($service_type) . '.inc.php';
```
Comment (Jellyfrog) [inline includes/services.inc.php:131]: this variable doesn't exist

Comment (murrant) [reply includes/services.inc.php]: Thanks

**Pull request merged**